iron-meta.html

```diff
@@ -57,12 +57,12 @@
 
     /**
      * @constructor
-     * @param {{type: (string|null), key: (string|null), value: *}} options
+     * @param {{type: (string|null)=, key: (string|null)=, value: *=}=} options
```

Comment:
This is wrong: it looks like the `=` suffix is only allowed on the types of parameters themselves and isn't just a shorthand for `| undefined`.